mtags/src/main/scala/scala/meta/internal/mtags/ClasspathLoader.scala

```diff
@@ -0,0 +1,30 @@
+package scala.meta.internal.mtags
+
+import scala.meta.io.AbsolutePath
+import scala.meta.io.Classpath
+import scala.meta.io.RelativePath
+
+/**
+ * Utility to load relative paths from a classpath.
+ *
+ * Provides similar functionality as URLClassLoader but does no caching.
```

Comment:
Is the no-caching a feature or a limitation? Maybe it's worth explaining why this class is needed over `URLClassLoader`

Reply:
Most important bit is that it uses `AbsolutePath` and `Option[T]` instead of `java.net.URL` and nulls. Updated docstring to reflect that.

Reply:
Lack of caching was actually out-dated, the underlying implementation uses URLClassLoader at the moment.